src/telemetry/index.ts

```diff
@@ -1526,6 +1526,7 @@ export interface IEventNamePropertyMapping {
     };
     [DebuggingTelemetry.clickedRunByLine]: never | undefined;
     [DebuggingTelemetry.successfullyStartedRunByLine]: never | undefined;
+    [DebuggingTelemetry.successfullyStartedIWJupyterDebugger]: never | undefined;
```

Comment:
Can you add a comment describing the telemetry? It's for the telemetry.md file.

Reply:
Next PR